packages/iTwinUI-react/src/core/Table/Table.test.tsx

```diff
@@ -352,54 +352,56 @@ it('should handle checkbox clicks', async () => {
   expect(onSelect).toHaveBeenCalledWith([mockedData()[1]], expect.any(Object));
 
   await userEvent.click(checkboxCells[0]);
-  expect(onSelect).toHaveBeenCalledWith(mockedData(), expect.any(Object));
+  expect(onSelect).toHaveBeenCalledWith([], expect.any(Object));
```

Comment:
This change must be reverted as it is a result of #864 which is in `main` and not in `v2`.

Reply:
**[DONE]** Ohh, right! Thanks, reverted